### PR TITLE
Add support for namespaces

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,3 +112,5 @@ To uninstall the chart:
 | endpointMonitor.enabled | Enable endpointMonitor for IMC (https://github.com/stakater/IngressMonitorController) | `false` |
 | endpointMonitor.additionalLabels | Labels for endpointMonitor | `{}` |
 | endpointMonitor.annotations | Annotations for endpointMonitor | `{}` |
+| namespace.create | Enable namespace | `false` |
```
Comment (rasheedamir) [inline README.md:115]: @hanzala1234 you mean enabled? and not create

File: application/templates/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{{- if .Values.deployment }}
+{{- if .Values.namespace.enabled -}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace.name }}
+{{- end -}}
```
Comment (rasheedamir) [inline application/templates/namespace.yaml:7]: @hanzala1234 add support to specify labels, annotations, etc.

File: README.md (hunk 1 above)
Comment (hanzala1234) [reply README.md]: Good Catch!
Done

File: application/templates/namespace.yaml (hunk 2 above)
Comment (hanzala1234) [reply application/templates/namespace.yaml:17]: Done

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**